server/tests/db-entry-mockdata.js

```diff
@@ -13,23 +13,31 @@ const entries = {
     password: 'Passwo$#89rd@123',
```

Comment:
Are you dropping and then creating tables for the test.

Reply:
yeah, am deleting all data in tables and inserting new ones.